RiotSwiftUI/Modules/Room/LocationSharing/MapViewErrorAlertInfoBuilder.swift

```diff
@@ -18,28 +18,24 @@ import Foundation
 
 struct MapViewErrorAlertInfoBuilder {
     
-    func build(with error: LocationSharingViewError, dimissalCallback: (() -> Void)?) -> AlertInfo<LocationSharingAlertType>? {
+    func build(with error: LocationSharingViewError, primaryButtonCompletion: (() -> Void)?) -> AlertInfo<LocationSharingAlertType>? {
         
         let alertInfo: AlertInfo<LocationSharingAlertType>?
         
         switch error {
         case .failedLoadingMap:
             alertInfo = AlertInfo(id: .mapLoadingError,
                                                  title: VectorL10n.locationSharingLoadingMapErrorTitle(AppInfo.current.displayName),
-                                                 primaryButton: (VectorL10n.ok, dimissalCallback))
+                                                 primaryButton: (VectorL10n.ok, primaryButtonCompletion))
```

Comment:
I think "dismissalCallBack" is more generic as this action is used whenever it's a primaryButton or not